package.json

```diff
@@ -30,14 +30,14 @@
   "dependencies": {},
   "peerDependencies": {
     "axe-core": "^3 || ^4",
-    "cypress": "^3 || ^4 || ^5 || ^6"
+    "cypress": "^3 || ^4 || ^5 || ^6 || ^7"
```

Comment:
I'm wondering if this library would ever not be supported on future releases from v3?

Should it be just `>= 3`?

Reply:
Just tested with Cypress 8 and everything seems to be working well